dogfood/contents/main.tf

```diff
@@ -84,6 +85,30 @@ data "coder_parameter" "region" {
   }
 }
 
+data "coder_parameter" "res_mon_memory_threshold" {
+  type        = "number"
+  name        = "Memory usage threshold"
+  default     = 80
+  description = "The memory usage threshold used in resources monitoring to trigger notifications."
+  mutable     = true
+}
+
+data "coder_parameter" "res_mon_volume_threshold" {
+  type        = "number"
+  name        = "Volume usage threshold"
+  default     = 80
+  description = "The volume usage threshold used in resources monitoring to trigger notifications."
+  mutable     = true
+}
+
+data "coder_parameter" "res_mon_volume_path" {
+  type        = "string"
+  name        = "Volume path"
+  default     = "/home/coder"
+  description = "The path monitored in resources monitoring to trigger notifications."
+  mutable     = true
+}
```

Comment:
I don't see a good use of exposing the options to add your own monitor paths and thresholds.

Reply:
I see @dannykopping's comment now.